GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Engines.cfg

```diff
@@ -966,25 +966,17 @@
 }
 
 //  ==================================================
-//  Separation motor (medium).
+//  Radial Separation motor (medium).
 //  ==================================================
 
 @PART[sepMotor1]:FOR[RealismOverhaul]
 {
 	%RSSROConfig = True
-	@title = Separation Motor (Medium)
+	@title = Radial Separation Motor (Medium)
 	@description = Small solid motor use to help separate one stage from another or perform ullage. Best used with others.
 
 	@mass = 0.044
 
-	@MODULE[ModuleEngines*]
-	{
-		@name = ModuleEnginesRF
-		@maxThrust = 98
-		@heatProduction = 17.5
-		%exhaustDamage = False
```

Comment:
This one line you might still need?  Not sure if that specifically gets set by the MEC.

Reply:
The `%exhaustDamage`?

Reply:
Resolved in https://github.com/KSP-RO/RealismOverhaul/pull/2454/commits/274bf8c18023363735347e4114908b902535abbb